cli/flox/src/commands/install.rs

```diff
@@ -232,16 +270,38 @@ impl Install {
             message::warning(warning);
         }
 
-        // Print which new packages were installed
-        for pkg in packages_to_install.iter() {
-            if let Some(false) = installation.already_installed.get(pkg.id()) {
-                message::package_installed(pkg, &description);
-            } else {
-                message::warning(format!(
-                    "Package with id '{}' already installed to environment {description}",
-                    pkg.id()
-                ));
-            }
+        let installed = if let Some(packages_retried) = packages_retried {
+            packages_retried
+        } else {
+            packages_to_install
+                .iter()
+                .map(|p| PackageToInstallRetry {
+                    partial: false,
+                    pkg: p.clone(),
+                })
+                .collect::<Vec<_>>()
+        };
+        let partitioned =
+            Self::partition_installed_packages(&installed, &installation.already_installed);
+
+        // Print status messages for the installation attempt
+        if !partitioned.successes.is_empty() {
+            message::packages_successfully_installed(&partitioned.successes, &description);
+        }
+        for pkg in partitioned.partials.iter() {
+            message::warning(format!(
+                "'{}' installed only for the following systems: {}",
+                pkg.id(),
+                // Only `None` for flakes, which can't reach this code
+                // path anyway.
+                pkg.systems().unwrap_or_default().join(", ")
+            ))
+        }
+        for pkg in partitioned.already_installed.iter() {
+            message::warning(format!(
+                "Package with id '{}' already installed to environment {description}",
```

Comment:
Can you combine these into a comma separated list for consistency with successful installs?

(it still feels right to split the partial installs though because the available systems may be different for each)

Reply:
Sure, that's just something we hadn't talked about.